scripts/setup/ubuntu/install_cbmc.sh

```diff
@@ -15,7 +15,7 @@ fi
 UBUNTU_VERSION=$(lsb_release -rs)
 MAJOR=${UBUNTU_VERSION%.*}
 
-if [[ "${MAJOR}" -gt "18" ]]
+if [[ "${MAJOR}" -gt "18" ]] && [[ $(uname -m) = "x86_64" ]]
```

Comment:
I have to say... my ubuntu prints an empty line when I execute `uname -m`

Reply:
Well, that's surprising. Do you think `uname` is not reliable enough?
AFAIK `arch` is an alternative, but funnily enough it produces `i386` on my 2019 Macbook Pro 😬

Reply:
Yes, that's what I've been thinking. It will install from source at least on my machine every time I run this script. :disappointed: Anyway, it's likely that my machine is a bit messed up, so I can create a little workaround for me.

Reply:
Since we're on ubuntu anyway, perhaps we should use `dpkg --print-architecture`?

Reply:
Since we're on ubuntu anyway, perhaps we should use `dpkg --print-architecture`?

Reply:
I tested on both aarch64 and amd64, and `dpkg --print-architecture` works fine. Changed to that, hopefully this is reliable (I think it should be, as that's what we care about anyway: we want to install a amd64 deb so dpkg arch should be amd64)

Reply:
Thank you @bennofs for finding and testing an alternative!